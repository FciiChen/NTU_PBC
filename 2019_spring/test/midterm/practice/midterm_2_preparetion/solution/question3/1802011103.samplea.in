midterm2_q3_ds1.csv