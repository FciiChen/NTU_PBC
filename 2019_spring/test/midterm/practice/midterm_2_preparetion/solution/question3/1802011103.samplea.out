name2,160
name4,88,8888
name5,This
name99,Hello,World,is
